persistence/src/main/java/de/cyface/persistence/serialization/MeasurementSerializer.java

```diff
@@ -73,7 +73,7 @@
  *
  * @author Klemens Muthmann
  * @author Armin Schnabel
- * @version 4.1.0
+ * @version 4.2.0
```

Comment:
As this is a non-trivial  change (could break something somewhere) I decided to increase the 2nd number.